plugins/3-django.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+test -f manage.py || return 1
+
+django_server() {
+  run python manage.py server
```

Comment:
This should be `python manage.py runserver`
https://docs.djangoproject.com/en/2.0/ref/django-admin/#runserver